x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/assistant/assistant_card.tsx

```diff
@@ -34,10 +44,12 @@ export const AssistantCard: OnboardingCardComponent<AssistantCardMetadata> = ({
   const connectors = checkCompleteMetadata?.connectors;
 
   return (
-    <OnboardingCardContentPanel style={{ paddingTop: 0 }}>
+    <OnboardingCardContentPanel
+      style={{ paddingTop: 0, backgroundColor: isDarkMode ? euiTheme.colors.lightestShade : '' }}
```

Comment:
Can we avoid assigning `''` to the `backgroundColor`?
```suggestion
      style={{
        paddingTop: 0,
        ...(isDarkMode && { backgroundColor: euiTheme.colors.lightestShade }),
      }}
```